src/wh_server.c

```diff
@@ -28,8 +30,573 @@
 
 #include "wolfhsm/wh_message.h"
 #include "wolfhsm/wh_message_comm.h"
+#include "wolfhsm/wh_packet.h"
 #include "wolfhsm/wh_server.h"
 
+#ifdef WOLFHSM_SHE_EXTENSION
```

Comment:
Recommend to move all of the SHE stuff to a separate file.  Ensure the any state variables are members of the server context to avoid globals.